packages/expect/src/print.ts

```diff
@@ -66,6 +66,32 @@ export const printReceivedArrayContainExpectedItem = (
       ']',
   );
 
+export const printCloseTo = (
+  receivedDiff: number,
+  expectedDiff: number,
+  precision: number,
+  isNot: boolean,
+): string => {
+  const receivedDiffString = stringify(receivedDiff);
+  const expectedDiffString = receivedDiffString.includes('e')
+    ? expectedDiff.toExponential(0)
+    : 0 <= precision && precision <= 20
```

Comment:
Maybe a comment for the magic numbers? I had to look up `toFixed` on MDN to understand it.

Reply:
Yes, especially because `20` is off by one because of `precision + 1`